test/ids.t

```diff
@@ -48,6 +48,10 @@ class TestIds(TestCase):
         self.assertIn(' @1 ', out)
         self.assertIn(' @2 ', out)
 
+    def test_should_fail_on_zero_id(self):
+        code, out, err = self.t.runError("delete @0")
+        self.assertIn("IDs must be specified.", err)
```

Comment:
Could we use an error message like "@0 is not a valid ID"? I think it addresses the problem better as an ID *is* specified, it is just not valid...

Reply:
Yes, that's a good idea. Will do.